unity-environment/Assets/ML-Agents/Examples/Crawler/Scripts/CrawlerAgent.cs

```diff
@@ -0,0 +1,311 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CrawlerAgent : Agent {
+    [Header("Target To Walk Towards")] 
+    [Space(10)] 
+    public Transform target;
+    public bool respawnTargetWhenTouched;
+    public float targetSpawnRadius;
+
+
```

Comment:
Too many blank lines

Reply:
fixed

Reply:
I think the formatting is inconsistent. Could you reformat all the .cs scripts ? (for example new line for bracket)